src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -476,9 +474,10 @@ private void AddGroupToGroup(object obj)
             {
                 var selectedModels = DynamoSelection.Instance.Selection
                     .OfType<AnnotationModel>()
-                    .Where(x => x.GUID != this.AnnotationModel.GUID && 
+                    .Where(x => x.GUID != this.AnnotationModel.GUID &&
                                 !WorkspaceViewModel.Model.Annotations.ContainsModel(x));
 
+                using (NestedGroupsGeometryCollection.DeferCollectionReset())
```

Comment:
this is hard for me to follow without `{` `}` - I guess it applies to the for loop, but I'd prefer not to guess.